extensions/amp-story-interactive/0.1/test/validator-amp-story-interactive-valid.html

```diff
@@ -0,0 +1,89 @@
+<!--
```

Comment:
Please label which ones are validnad which ones are invalid and why. E.g. before the html segment that tests each condition include `<!-- Valid: amp story interactive poll -->`, `<!-- Invalid: invalid poll due to missing ... -->`

Reply:
In this file all the components are valid (hence the name of the file), and the IDs of the elements are "correct-results" meaning it is valid. On the other validator tests the ID of the elements say what is wrong or right. Eg: missing-endpoint-quiz -> the quiz needs an "endpoint" that was not specified. If the ID is correct-quiz, then it's valid

Reply:
Please update the test to include these comments for each individual test.

Reply:
Done!